funcframework/framework.go

```diff
@@ -37,20 +39,35 @@ const (
 	errorStatus          = "error"
 )
 
+var (
+	handler = http.DefaultServeMux
+)
+
 // RegisterHTTPFunction registers fn as an HTTP function.
 func RegisterHTTPFunction(path string, fn interface{}) {
 	fnHTTP, ok := fn.(func(http.ResponseWriter, *http.Request))
 	if !ok {
-		panic("expected function to have signature func(http.ResponseWriter, *http.Request)")
+		log.Fatalf("expected function to have signature func(http.ResponseWriter, *http.Request), got %s", reflect.TypeOf(fn))
 	}
-	registerHTTPFunction(path, fnHTTP, http.DefaultServeMux)
+	registerHTTPFunction(path, fnHTTP, handler)
 }
 
 // RegisterEventFunction registers fn as an event function. The function must have two arguments, a
 // context.Context and a struct type depending on the event, and return an error. If fn has the
-// wrong signature, RegisterEventFunction panics.
+// wrong signature, RegisterEventFunction logs a fatal error.
 func RegisterEventFunction(path string, fn interface{}) {
-	registerEventFunction(path, fn, http.DefaultServeMux)
+	registerEventFunction(path, fn, handler)
+}
+
+// RegisterCloudEventFunction registers fn as an cloudevent function. The function must have one
+// arguments, a cloudevents.CloudEvent, and return an error. If fn has the wrong signature,
```

Comment:
Doesn't look like the function should return an error?

Reply:
Updated comment.